operator-common/src/main/java/io/strimzi/operator/common/Util.java

```diff
@@ -543,13 +543,24 @@ public static boolean isMaintenanceTimeWindowsSatisfied(Reconciliation reconcili
     }
 
     /**
-     * Encodes a String into Base64
+     * Encodes a String into Base64.
      *
-     * @param encode    String which should be encoded
+     * @param data String that should be encoded.
      *
-     * @return          Base64 data
+     * @return Base64 data.
      */
-    public static String encodeToBase64(String encode)  {
-        return Base64.getEncoder().encodeToString(encode.getBytes(StandardCharsets.US_ASCII));
+    public static String encodeToBase64(String data)  {
+        return Base64.getEncoder().encodeToString(data.getBytes(StandardCharsets.US_ASCII));
+    }
+
+    /**
+     * Decodes a String from Base64.
+     *
+     * @param data String that should be decoded.
+     *
+     * @return Plain data.
+     */
+    public static String decodeFromBase64(String data)  {
```

Comment:
I do not have a problem with adding this method. But you should probably open an issue (do not do it in this PR) to have it used from the modules that already depend on `operator-common` as there seem to be many uses of this across the code.

Reply:
I can open an issue to follow up on this task right after this is merged, but I would like to keep it. In the same PR, we can also evaluate if we should switch from ASCII to UTF8. How does that sounds?

Reply:
Opened #9745 and #9746 to discuss them separately.